<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConversionTools</class>
 <widget class="QWidget" name="ConversionTools">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>552</width>
    <height>418</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>552</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>552</width>
    <height>418</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>数据转换工具</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/images/favicon.ico</normaloff>:/images/favicon.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="HexToDecBtn">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>80</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Hex-&gt;Dec</string>
   </property>
  </widget>
  <widget class="QPushButton" name="DecToHexBtn">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>110</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Dec-&gt;Hex</string>
   </property>
  </widget>
  <widget class="QPushButton" name="HexToAscBtn">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>140</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Hex-&gt;Ascii</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AscToHexBtn">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>170</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Ascii-&gt;Hex</string>
   </property>
  </widget>
  <widget class="QPushButton" name="HexToFloatBtn">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>200</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Hex-&gt;Float</string>
   </property>
  </widget>
  <widget class="QPushButton" name="FloatToHexBtn">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>230</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Float-&gt;Hex</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CharacterNumBtn">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>380</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>字符个数</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ToOppositeBtn">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>200</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>转换为反码</string>
   </property>
  </widget>
  <widget class="QPushButton" name="RandomNumBtn">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>380</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>随机数</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ToUpperBtn">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>140</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>转换为大写</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ToLowerBtn">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>170</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>转换为小写</string>
   </property>
  </widget>
  <widget class="QPushButton" name="InversionBtn">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>230</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>字符串反转</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BitwiseAndBtn">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>20</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>按位与</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BitwiseOrBtn">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>50</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>按位或</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BitwiseXorBtn">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>80</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>按位异或</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BitwiseNotBtn">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>110</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>按位取反</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>10</y>
     <width>91</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>输入格式</string>
   </property>
   <widget class="QRadioButton" name="AscRadioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ascii</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="HexRadioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Hex</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="inputlabel1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>输入1：</string>
   </property>
  </widget>
  <widget class="QLabel" name="inputlabel2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>输入2：</string>
   </property>
  </widget>
  <widget class="QLabel" name="outputlabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>输出：</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>341</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="input1Layout"/>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>341</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="input2Layout"/>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>341</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="outputLayout"/>
  </widget>
  <widget class="QPushButton" name="GbkToChBtn">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>GBK-&gt;汉字</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Utf8ToChBtn">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>290</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>UTF8-&gt;汉字</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ChToGbkBtn">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>汉字-&gt;GBK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ChToUtf8Btn">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>290</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>汉字-&gt;UTF8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="GbkToUtf8Btn">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>320</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>GBK-&gt;UTF8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Utf8ToGbkBtn">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>320</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>UTF8-&gt;GBK</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
